src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -61,14 +61,8 @@ public enum WriteMode {
     REST_API,
   }
 
-  /** The write mode to generate Arrow BDEC file. */
+  /** The write mode to generate BDEC file. */
   public enum BdecVersion {
-    /** Uses Arrow to generate BDEC chunks. */
-    ONE(1),
```

Comment:
I would consider to comment it out for history, just to see what we used prev numbers for, like `TWO(2)`

Reply:
Commented out